simphony/cuds/abc_modeling_engine.py

```diff
@@ -22,6 +25,31 @@ class ABCModelingEngine(object):  # pragma: no cover
     """
     __metaclass__ = ABCMeta
 
+    def __init__(self, *args, **kwargs):
```

Comment:
> In addition, even without putting the arguments into CUDS, we would still have the issue with the docstring of Simulation.__init__: A user new to the framework will need to use get_wrapper(wrapper_name) in order to fetch the class and its docstring.

My above comment regarding fetching the docstring is referring to the situation when `*args` and `**kwargs` are allowed in `Simulation.__init__` (otherwise I don't know how to comply with `ABCModelingEngine.__init__(self, *args, **kwargs)`.  If all the required parameters (e.g. use_internal_interface) are to be moved to CUDS, then `ABCModelingEngine.__init__(self, *args, **kwargs)` would have been `ABCModelingEngine.__init__(self)` and `Simulation.__init__` accepts no more `*args` or `**kwargs` (as it is now).  Have I understood the intention correctly?